platform/podjs/rollup.config.js

```diff
@@ -1,48 +1,58 @@
 import commonjs from "@rollup/plugin-commonjs";
-import resolve from "@rollup/plugin-node-resolve";
 import sucrase from "@rollup/plugin-sucrase";
 import json from "@rollup/plugin-json";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import merge from "deepmerge";
+import { createBasicConfig } from "@open-wc/building-rollup";
+import path from "path";
 
-export default [
```

Comment:
I can't quite make sense of these changes - in my book, the Rollup configuration of _podjs_ doesn't need to be changed at all, since it still does exactly what it did before, no?

Reply:
yeah these are some leftovers from the rollup-plugins tries that I was giving...thought to leave them as they are harmless

Reply:
Isn't it unused dependencies code? Not a big fan of adding unused code, I wouldn't call that _harmless_ :D Makes it harder to figure out which part of the code base actually does something and which part is just there doing nothing.

Reply:
Me neither, unfortunately. I'd better leave them as originally. I have the hunch that the changes were not totally reverted, leading to a total overhaul of all package-lock.json. As a matter of fact, the change of package-lock.json here without a corresponding change in package.json is kind of a giveaway.

Reply:
this is already reverted

Reply:
I referred to the changes in rollup.config. I've re-run all the `npm i` and still bring changes on the `lock` files though :/

Reply:
Yep, they're upgrading some stuff if it becomes available. I would really feel more comfortable if you checked them in from main to make them the same, if there's no real related change in here. If that breaks something (it shouldn't) I'd have to take a look at the lockfiles anyway, although they're probably OK.